### PR TITLE
Implement the Commerce\Orders::find_local_order() method

### PR DESCRIPTION
**Pull request by deivamagalhaes**
# Summary

This PR implements the `Commerce\Orders::find_local_order()` method.

## QA

- [x] Integrations test pass

## Before merge

- [ ] I have confirmed these changes in each supported minor WooCommerce version

### CODE REVIEW

File: includes/Commerce/Orders.php [hunk 1]
```diff
@@ -40,8 +40,21 @@ class Orders {
 	 */
 	public function find_local_order( $remote_id ) {
 
-		// TODO: implement
-		return null;
+		$order_id_array = get_posts( [
```
Comment (ChaseWiseman) [inline includes/Commerce/Orders.php:43]: How about we use [`wc_get_orders()`](https://github.com/woocommerce/woocommerce/wiki/wc_get_orders-and-WC_Order_Query)? That way we'll have less to update once WC switches away from custom post types. It seems to still support the same meta query args.

File: includes/Commerce/Orders.php [hunk 2]
```diff
@@ -69,10 +82,84 @@ public function create_local_order( Order $remote_order ) {
 	 * @param Order $remote_order Orders API order object
 	 * @param \WC_Order $local_order local order object
 	 * @return \WC_Order
+	 * @throws SV_WC_Plugin_Exception|\WC_Data_Exception
 	 */
 	public function update_local_order( Order $remote_order, \WC_Order $local_order ) {
 
-		// TODO: implement
+		// add/update items
+		foreach ( $remote_order->get_items() as $item ) {
```
Comment (ChaseWiseman) [inline includes/Commerce/Orders.php:90]: It looks like this PR and #1515 contain the same changes, both to `find_local_order()` and `update_local_order()`. Can we get the two isolated to their respective methods?

Comment (deivamagalhaes) [reply includes/Commerce/Orders.php]: Weird. I force-pushed this one to the correct commit.

File: includes/Commerce/Orders.php (hunk 1 above)
Comment (deivamagalhaes) [reply includes/Commerce/Orders.php]: Updated in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1514/commits/b6e5820a5b270a5cea1b187a24eca56984c80bea

**Pull request merged**